src/Bedrock.Framework.Experimental/Protocols/Memcached/MemcachedRequest.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using static Bedrock.Framework.Experimental.Protocols.Memcached.Enums;
+
+namespace Bedrock.Framework.Experimental.Protocols.Memcached
+{
+    public class MemcachedRequest
+    {
+        public Opcode Opcode { get; }
+        public byte[] Key { get; }
+        public uint Opaque { get; }
```

Comment:
Do we want `uint` in the public signature?

I don't care, but in respect to CLS -- I'm just asking.

Reply:
Doesn't matter right now